package.json

```diff
@@ -108,6 +108,7 @@
     "react-password-strength": "^2.4.0",
     "react-redux": "^5.1.1",
     "react-router-dom": "^4.3.1",
+    "recompose": "^0.30.0",
```

Comment:
packaging PR please?

Reply:
https://github.com/theforeman/foreman-packaging/pull/3825

Reply:
btw where is this used?

Reply:
not used yet here, only in the blocked PRs

Reply:
hmm.. taken from https://github.com/acdlite/recompose

```
A Note from the Author (acdlite, Oct 25 2018):
Hi! I created Recompose about three years ago. About a year after that, I joined the React team. Today, we announced a proposal for Hooks. Hooks solves all the problems I attempted to address with Recompose three years ago, and more on top of that. I will be discontinuing active maintenance of this package (excluding perhaps bugfixes or patches for compatibility with future React releases), and recommending that people use Hooks instead. Your existing code with Recompose will still work, just don't expect any new features. Thank you so, so much to @wuct and @istarkov for their heroic work maintaining Recompose over the last few years.

Read more discussion about this decision here.
```